src/I18n/TranslatorRegistry.php

```diff
@@ -76,7 +76,7 @@ public function setCacher(CacheEngine $cacher)
      * @param string $name The translator package to retrieve.
      * @param string|null $locale The locale to use; if empty, uses the default
      * locale.
-     * @return \Aura\Intl\TranslatorInterface A translator object.
+     * @return \Aura\Intl\TranslatorInterface|null A translator object.
```

Comment:
Would be nice if that was removed in 4.0 then. A get should never return just null - ideally always an object, or a Null object of some sort. This is a hidden little no op issue here, and the doc block at least now shows the possible null value. Blindly using it without checking for existence doesn't sound like a safe idea right now :)